contracts/BondToken.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+import "./HubPoolInterface.sol";
+import "./WETH9.sol";
+
+interface _HubPool is HubPoolInterface {
```

Comment:
Prefer sticking to existing naming convention and not prefixing this with a _. Maybe name HubPoolExtendedInterface or something, or even IHubPool

Reply:
Good point - I'll find something better :+1:

Reply:
Implemented: https://github.com/across-protocol/contracts-v2/pull/226/commits/91c1780c121f2782eed0e552092c6f017cf23f42